src/providers/twitch/TwitchChannel.cpp

```diff
@@ -836,6 +836,20 @@ void TwitchChannel::loadRecentMessages()
                 }
             }
 
+            if (!getSettings()->grayOutHistory)
+            {
+                const auto str =
+                    QString(
+                        "[​ ​ ​ ​ ​ ​ ​ ​ ​ ​ ​ ​ ​ "
+                        "​ ​ ​ ​ ​ ​ ​ ​ ​ ▲ Chat history "
+                        "before %1 %2 ▲ ​ ​ ​ ​ ​ ​ ​ ​ ​ "
+                        "​ ​ ​ ​ ​ ​ ​ ​ ​ ​ ​ ​ ​]")
+                        .arg(QDateTime::currentDateTime().toString("dd/MM/yy"),
```

Comment:
I'd suggest you just set the timestamp on this message instead of using a hardcoded date format. As for the spaces, please use the `Center` `MessageFlag` instead if you for the center effect. Using spaces is inconsistent because Chatterino allows setting non-monospace fonts. It can also look bad with small width-wise windows when it's wrapping to multiple lines.

Consider using [this ↓](https://github.com/Chatterino/chatterino2/blob/master/src/messages/MessageBuilder.cpp#L26) version of `makeSystemMessage` to set the timestamp in the generated message.
```cpp
MessagePtr makeSystemMessage(const QString &text, const QTime &time);
```

Reply:
I'll go ahead and make these changes, the hard coded string I have used was taken from one of the suggestions in the issue so i thought it was fine to use that.

Reply:
Also since we remove the hard coded date format from the string, in my opinion the message displayed should also change as "Chat history before" does not make much sense anymore. Any suggestions?